netweaver/monitoring.sls

```diff
@@ -20,17 +23,37 @@ sap_host_exporter_configuration_{{ exporter_instance }}:
          sap-control-uds: /tmp/.sapstream5{{ sap_instance_nr }}13
     - require:
       - pkg: prometheus_sap_host_exporter_pkg
-      - netweaver_install_{{ instance_name }}
+# on HA use case deploy ASCS|ERS on ERS|ASCS
+{% if netweaver.ha_enabled and node.sap_instance in ["ascs", "ers"] %}
+    - onlyif:
+      - test -d /usr/sap/{{ node.sid }}/ASCS* || test -d /usr/sap/{{ node.sid }}/ERS*
```

Comment:
I would put maybe here some `node.sid.upper()` (or the method to uppercase the string) just in case

Reply:
@arbulu89 fixed in fbc9d8e